pos_product_multi_barcode/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,4 @@
+* Ooops404 <https://ooops404.com>
```

Comment:
funny that the original author @PierrickBrun is not here... can you fix this pls?